src/material/snack-bar/testing/snack-bar-harness.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness} from '@angular/cdk/testing';
+
+/**
+ * Harness for interacting with a standard mat-snack-bar in tests.
+ * @dynamic
+ */
+export class MatSnackBarHarness extends ComponentHarness {
+  // Developers can provide a custom component or template for the
+  // snackbar. The canonical snack-bar parent is the "MatSnackBarContainer".
+  static hostSelector = '.mat-snack-bar-container';
```

Comment:
add `with` method for consistency with our other harnesses

Reply:
Done.